common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -2719,6 +2719,33 @@
     ],
     "sqlState" : "42000"
   },
+  "INVALID_FLOW_RELATION_TYPE" : {
```

Comment:
Any particular reason to use "relation" instead of "query"? I think the latter is more common in docs.